inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -375,19 +375,6 @@ static void Transformation(CNNNetwork& clonedNetwork, const Config& conf) {
         std::string errMsg;
         return MKLDNNFakeQuantizeNode::isSupportedOperation(node, errMsg);
     });
-    postLPTPassManager.get_pass_config()->set_callback<ngraph::pass::AddMultiplyFusion>([](const_node_ptr &node) -> bool {
-        if (auto mul_op = std::dynamic_pointer_cast<const ngraph::opset1::Multiply>(node)) {
-            auto add_op = std::dynamic_pointer_cast<const ngraph::opset1::Add>(mul_op->get_input_node_shared_ptr(0));
-            auto constant = std::dynamic_pointer_cast<const ngraph::opset1::Constant>(mul_op->get_input_node_shared_ptr(1));
-            bool is_dequantization = mul_op->get_rt_info().count("DEQUANTIZATION") != 0;
-            if (add_op && constant && is_dequantization) {
-                return ngraph::is_type<ngraph::opset1::Convolution>(add_op->get_input_node_shared_ptr(0)) ||
```

Comment:
Don't we have performance degradations after removing this? Are biases still presence and not swap with dequantization multiplies?

Reply:
It's weird, but the AddMultiplyFusion is not even added to postLPTPassManager (we have only ngraph::pass::FakeQuantizeDecomposition and ngraph::pass::UnrollTensorIterator). To my mind, this callback looks like useless.

Reply:
It's really strange. This callback have to be in part of common optimizations. We need to investigate impact of this lines in common optimizations then.

Reply:
new tasks was created: 63613